physionet-django/project/views.py

```diff
@@ -1732,15 +1732,15 @@ def published_project_license(request, project_slug, version):
         'license_content':license_content})
 
 
-def published_project_required_trainings(request, project_slug, version):
+def published_project_required_training(request, project_slug, version):
     """Displays a published project's required training"""
     project = get_object_or_404(PublishedProject, slug=project_slug, version=version)
 
     required_trainings = project.required_trainings.all()
 
     return render(
         request,
-        'project/published_project_required_trainings.html',
+        'project/published_project_required_training.html',
         {'project': project, 'required_trainings': required_trainings},
```

Comment:
I think this line needs updating to match the new template?

`{'project': project, 'required_trainings': required_trainings}` should be:
 `{'project': project, 'required_training': required_trainings}`

Reply:
Thanks Tom. I will make the changes

Good Catch